packages/workflows/src/handlers/product/update-product-variants.ts

```diff
@@ -14,21 +14,22 @@ export async function updateProductVariants({
 > {
   const { productVariantsMap } = data
   const productsVariants: ProductTypes.UpdateProductVariantDTO[] = []
-  const updateProductsData: ProductTypes.UpdateProductDTO[] = []
+  const updateVariantsData: ProductTypes.UpdateProductVariantOnlyDTO[] = []
   const productModuleService: ProductTypes.IProductModuleService =
     container.resolve(ModulesDefinition[Modules.PRODUCT].registrationName)
 
-  for (const [productId, variantsData = []] of productVariantsMap) {
-    updateProductsData.push({
-      id: productId,
-      variants: variantsData,
-    })
+  for (const [product_id, variantsData = []] of productVariantsMap) {
```

Comment:
question: the variants data doesnt include the product id already?

Reply:
variantsData is the update for the variant, not the variant from the db, I will rename it for clarity 😄